RestaurantReservation.API/Controllers/EmployeeController.cs

```diff
@@ -0,0 +1,113 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using RestaurantReservation.API.Models.Employee;
+using RestaurantReservation.Db;
+
+namespace RestaurantReservation.API.Controllers
+{
+    [ApiController]
+    [Authorize(Policy = "Manager")]
+    [Route("api/restaurants/employees")]
```

Comment:
Why the restaurant ID parsed from the claims instead of having it in the endpoint? (`/api/restaurants/{restaurantId}/employees`)